sirepo/pkcli/service.py

```diff
@@ -52,25 +59,159 @@ def flower():
         ])
 
 
-def http():
-    """Starts Flask server in http mode.
+def flask():
+    from sirepo import server
+
+    use_reloader = pkconfig.channel_in('dev')
+    app = server.init(use_reloader=use_reloader)
+    # avoid WARNING: Do not use the development server in a production environment.
+    app.env = 'development'
+    app.run(
+        host=cfg.ip,
+        port=cfg.port,
+        threaded=True,
+        use_reloader=use_reloader,
+    )
+
+
+def http(driver='local', nersc_proxy=None, nersc_user=None, sbatch_host=None):
+    """Starts the Flask server and job_supervisor.
 
     Used for development only.
+
+    Args:
+        driver (string): The driver type to enable (one of local, docker, sbatch, nersc)
+        nersc_proxy (string): A proxy nersc can use to reach the supervisor
+        nersc_user(string): A nersc user ??
```

Comment:
etc/run-supervisor.sh has the ability to pass in the nersc_user. The nersc_user is [only used to find the sirepo command](https://github.com/radiasoft/sirepo/blob/master/sirepo/job_driver/sbatch.py#L126). So, I think it is always nagler. Is that right? If so do we need to have it as a param?